### PR TITLE
Add logs to msal-browser

### PR DESCRIPTION
**Pull request by jo-arroyo**
This PR:
- Adds log messages to BrowserCacheManager.ts
- Updates log messages in RedirectHandler.ts

### CODE REVIEW

File: change/@azure-msal-browser-3e900bbd-5fb6-4800-9cf5-beeaefbc5e47.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
```
Comment (tnorling) [inline change/@azure-msal-browser-3e900bbd-5fb6-4800-9cf5-beeaefbc5e47.json:2]:
```suggestion
  "type": "patch",
```

File: lib/msal-browser/src/cache/BrowserCacheManager.ts [hunk 2]
```diff
@@ -288,6 +295,7 @@ export class BrowserCacheManager extends CacheManager {
      * @param appMetadataKey
      */
     getAppMetadata(appMetadataKey: string): AppMetadataEntity | null {
+        this.logger.verbose("BrowserCacheManager.getAppMetadata called");
         const value = this.getItem(appMetadataKey);
         if (!value) {
             return null;
```
Comment (tnorling) [inline lib/msal-browser/src/cache/BrowserCacheManager.ts:301]: In each of the getters should we also log whether or not we successfully found something in the cache?

**Review by tnorling — APPROVED** [review]
One non-blocking comment. Can be done incrementally if that makes more sense.

File: lib/msal-browser/src/cache/BrowserCacheManager.ts (hunk 2 above)
Comment (pkanher617) [reply lib/msal-browser/src/cache/BrowserCacheManager.ts:301]: +1

Comment (jo-arroyo) [reply lib/msal-browser/src/cache/BrowserCacheManager.ts:304]: Makes sense. I added cache hit logs, and it makes the logs a lot longer. If we discover it creates too much noise, I'm happy to remove them later.

**Pull request merged**